### PR TITLE
Set core path for annotations explicitly

### PR DESCRIPTION
**Pull request by DE3000**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -194,12 +196,14 @@ project(":core"){
...
     task preGen{
-        outputs.upToDateWhen{ false } 
+        outputs.upToDateWhen{ false }
         generateLocales()
         writeVersion()
     }
 
     dependencies{
+		// Set corePath to current project folder
+		System.properties["corePath"] = file(Paths.get(".")).toString()
```
Comment (Anuken) [inline build.gradle]: Why not use ``file("").absolutePath`, or something similar? Importing `nio` doesn't seem necessary.

Comment (DE3000) [reply build.gradle]: Fixed.

**Comment by DE3000** [comment]
Changed.

**Comment by Anuken** [comment]
Merged manually in 982483f178de845f6aa02f0fccefd2f0290f288c.

**Pull request closed without merging**